go/inst/instance_topology.go

```diff
@@ -1675,6 +1677,33 @@ func TakeSiblings(instanceKey *InstanceKey) (instance *Instance, takenSiblings i
 	return instance, len(relocatedReplicas), err
 }
 
+// Created this function to allow a hook to be called after a successful TakeMaster event
+func TakeMasterHook(master *Instance, slave *Instance) {
+	masterInstance := master.Key
+	slaveInstance := slave.Key
+	env := goos.Environ()
+	env = append(env, fmt.Sprintf("ORC_SUCCESSOR_HOST=%s", masterInstance))
+	env = append(env, fmt.Sprintf("ORC_FAILED_HOST=%s", slaveInstance))
+	successor := fmt.Sprintf("%s", masterInstance)
+	oldmaster := fmt.Sprintf("%s", slaveInstance)
```

Comment:
this is a bit confusing. Who is `master` in this story?

Reply:
Sorry about that, I'm making the requested changes so it isn't confusing.

Reply:
I've pushed the changes, please let me know if this is ok. Thanks for all the help and input 😄  👍